test/s3select_test.cpp

```diff
@@ -2827,3 +2872,30 @@ phoneNumbers.addr. : 88
 
 }
 
+TEST(TestS3selectFunctions, csv_json_count)
+{
+  std::string input;
+  std::string result;
+  size_t size = 5;
+  generate_csv(input, size);
+  std::string js = convert_to_json(input.c_str(), input.size());
+  const char* input_query_1 = "select count(0) from s3object[*];";
+
+  run_json_query(input_query_1, js, result);
```

Comment:
the concept of converting CSV to JSON is about running 
the same query on the same input (the JSON input is a conversion from CSV)
and compared both results.
in that way, we gain coverage for all SQL operators.

as mentioned before the SQL statement also needs a conversion.